lib/unit_tests.html

```diff
@@ -11,8 +11,8 @@
     <title>Nessie Javascript SDK Tests</title>
     
     <!-- QUnit Required Files -->
-    <link rel="stylesheet" href="//code.jquery.com/qunit/qunit-1.20.0.css">
-    <script src="//code.jquery.com/qunit/qunit-1.20.0.js"></script>
+    <link rel="stylesheet" href="http://code.jquery.com/qunit/qunit-1.20.0.css">
+    <script src="http://code.jquery.com/qunit/qunit-1.20.0.js"></script>
```

Comment:
Also you need to run your tests using ```Firefox``` browser and a ```firebug``` debug tool. It helps to effectively debug future issues like this case.